.buildkite/scaling/pipeline.sh

```diff
@@ -64,22 +64,17 @@ done
 # set up environment and agents
 cat << 'EOM'
 agents:
-  queue: central
-  modules: julia/1.10.0 cuda/12.2 ucx/1.14.1_cuda-12.2 openmpi/4.1.5_cuda-12.2 nsight-systems/2023.3.1
+  queue: new-central
+  modules: climacommon/2024_02_27
+  slurm_constraint: epyc
```

Comment:
Please, add a comment to explain why we are setting a constraint

(And is there a specific reason you picked eypc over icelake?)

Reply:
I tried epyc because I tried to restrict the `JULIA_CPU_TARGET` to just icelake and I got the following error:
```julia
ERROR: Your CPU does not support the CX16 instruction, which is required by this version of Julia! 
This is often due to running inside of a virtualized environment. 
Please read https://docs.julialang.org/en/v1/devdocs/sysimg/ for more.
```
I have set it back to icelake with the full cpu target.

Reply:
I think epyc is better for this particular test. Eypc nodes have 64 cores, and your scaling tests are powers of 2. icelake nodes have 56 cores.

I was asking if you picked epyc for any (other) particular reason.

Reply:
Ah, that is good to know, I will switch it back to epyc. I think I am misunderstanding how `JULIA_CPU_TARGET` works. 
Why do we need to supply targets besides epyc if the scaling pipeline is only running on epyc nodes?

Reply:
You shouldn't have to. I think you should even be able to remove the JULIA_CPU_TARGET line.

Reply:
There are only 28 epyc nodes and 462 icelake nodes and the scaling build has been waiting for 1 hour for epyc nodes, I think it makes sense to just use icelake even if the scaling tests are power of 2.

Reply:
In this case, I think we should change the number of cores we are testing over to make sure that we are properly saturating the node (in fact, typical scaling tests are for number of nodes, not for number of cores).